content/docs/engine-dev/tools/tiaf/maintaining-tiaf/debugging-tiaf.md

```diff
@@ -77,47 +87,182 @@ In the log above, you can extract the following information about the AR run and
+
+For this set of instructions, you will recreate the historic data locally from scratch.
+
+Checkout and build the commit used by the last run (in this instance `4b65521188a2a0beb399eca7b692216d2fe3d209`). For native runs, it is recommended you build the `debug` configuration to make debugging easier but for Python runs you must build the `profile` configuration as, at the time of writing, Python tests will not execute properly with the `debug` build configuration.
+
+Run the TIAF using the AR scripts to generate the historic data used in the failing AR run locally. To do so, run the [tiaf_driver.py](https://github.com/o3de/o3de/blob/development/scripts/build/TestImpactAnalysis/tiaf_driver.py) script and supply the arguments used in the AR run. For this demonstration, Visual Studio Code will be used. 
+
+The script configuration file used for this step is as follows:
+
+```json
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Current File",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "console": "integratedTerminal",
+            "justMyCode": true,
+            "args": [
+                "--config", "C:/dev/o3de/build/windows_vs2019/bin/TestImpactFramework/debug/Persistent/tiaf.json", // Replace with your build path to tiaf.json
+                "--src-branch", "development", // src and dst must be same in order to write out historic data
+                "--dst-branch", "development", // Note: the src and dst branches can be any name and do not need to refer to real, existing branches
+                "--commit", "4b65521188a2a0beb399eca7b692216d2fe3d209",
+                "--mars-index-prefix", "o3de-tiaf",
+                "--build-number", "001", // The build number can be anything
+                "--suites", "smoke", "main", // The suites must match those used in AR
+                "--label-excludes", "REQUIRES_gpu", // The label exclude filters must match those used in AR
+                "--test-failure-policy", "continue",
+                "--target-output", "stdout",
+                "--runtime-type", "native" // change to "python" for Python TIAF
+            ]
+        }
+    ]
+}
+```
+
+When the script has completed execution you will find the same historic data used in AR at the location `<build_path>/bin/TestImpactFramework/debug/Persistent/Native/historic/main-smoke`.
```

Comment:
```suggestion
Add the previously mentioned **Step 3**
```suggestion
3. Run the TIAF script to locally generates the historic data of the failing AR run.

After the script finishes executing, you can find the AR's historic data in the folder `<build_path>/bin/TestImpactFramework/debug/Persistent/Native/historic/main-smoke`. This local historic data is the same as the one from the Jenkins build you obtained in the [Prerequisite](#prerequisite) step.
```
```